src/main/java/net/snowflake/client/jdbc/SnowflakeDatabaseMetaData.java

```diff
@@ -184,6 +184,7 @@ private String unescapeChars(String escapedString) {
     String unescapedString = escapedString.replace("\\_", "_");
     unescapedString = unescapedString.replace("\\%", "%");
     unescapedString = unescapedString.replace("\\\\", "\\");
+    unescapedString = unescapedString.replace("\"", "\"\"");
```

Comment:
This fixes the double-quote problem by adding in the additional quote that is needed for SQL querying with a schema name containing quotes. 
Previous sample show command (incorrect): `show /* JDBC:DatabaseMetaData.getTables() */ objects in schema "TESTDB"."TEST_SCHEMA_"WITH_QUOTES""`
Current show command (correct): show /* JDBC:DatabaseMetaData.getTables() */ objects in schema "TESTDB"."TEST_SCHEMA_""WITH_QUOTES"""

Reply:
Is there any risk that we replace the undesired double quotes as well?

Reply:
No, there are no double quotes that would be present in the argument unless the user entered the schema pattern incorrectly. This is directly modifying the input argument.